src/app/src/components/AnimatedArcsAndMap/AnimatedArcsOverStates.tsx

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useRef, useCallback } from 'react';
+import { useMap } from 'react-leaflet';
+
+import L from 'leaflet';
+import '@elfalem/leaflet-curve';
+import { StateGeometry, StateProperties } from '../states.geojson';
+
+import StatesLayer from '../StatesLayer';
+import {
+    MonthlySpendingOverTime,
+    MonthlySpendingOverTimeByState,
+} from '../../types/api';
+import { MONTHLY_TIME_DURATION, DC_CENTER } from '../../constants';
+
+function getBezierOffsetLatLng(end: L.LatLngTuple): L.LatLngTuple {
+    // Modified Bezier curve function from https://gist.github.com/ryancatalani/6091e50bf756088bf9bf5de2017b32e6
+    const start: L.LatLngTuple = DC_CENTER;
+    var offsetX = end[1] - start[1],
+        offsetY = end[0] - start[0];
+
+    var r = Math.sqrt(Math.pow(offsetX, 2) + Math.pow(offsetY, 2)),
+        theta = Math.atan2(offsetY, offsetX);
+
+    var thetaOffset = 3.14 / 10;
+
+    var r2 = r / 2 / Math.cos(thetaOffset),
+        theta2 = theta + thetaOffset;
+
+    var midpointX = r2 * Math.cos(theta2) + start[1],
+        midpointY = r2 * Math.sin(theta2) + start[0];
+
+    return [midpointY, midpointX];
+}
+
+export default function AnimatedArcsOverStates({
+    animationEnabled,
+    spending,
+    totalTimeSteps,
+}: {
+    animationEnabled: boolean;
+    spending: MonthlySpendingOverTimeByState;
+    totalTimeSteps: number;
+}) {
+    const map = useMap();
+    const arcPathsReference = useRef<{ shape_code: string; curve: L.Curve }[]>(
+        []
+    );
+
+    !map.getPane('arcPathsPane') && map.createPane('arcPathsPane');
+
+    const createArcPath = useCallback(
+        (event: any) => {
+            const state = event.sourceTarget.feature.properties.STUSPS;
+            if (
+                arcPathsReference.current.find(
+                    path => path.shape_code === state
+                )
+            ) {
+                return;
+            }
+            const polygonCenter = (event.sourceTarget as L.Polygon)
+                .getBounds()
+                .getCenter();
```

Comment:
Can we use the pole of inaccessibility that Jacob calculated for the markers here? Perhaps in a follow up card if it seems not trivial.

Reply:
Yes! This ended up being easy to implement and is in [a3000da](https://github.com/azavea/green-equity-demo/pull/75/commits/a3000da312b262bfca14a519605560c7f4d398ce)